### PR TITLE
Release 16 with master

### PR DESCRIPTION
**Pull request by npauzenga**
**Summary:** Summary of changes

Addresses [CUMULUS-XX: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-XXX)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,23 @@ All notable changes to this project will be documented in this file.
...
+## [v16.0.0] 2023-05-09
+
+### MIGRATION notes
+
+#### PI release version
+
+When updating directly to v16 from prior releases older that V15, please make sure to
+read through all prior release notes.
+
+Notable migration concerns since the last PI release version (11.1.x):
```
Comment (Jkovarik) [inline CHANGELOG.md:18]: Wasn't the last PI release version v 14?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,23 @@ All notable changes to this project will be documented in this file.
...
+
+- [v14.1.0] - Postgres compatibility update to Aurora PostgreSQL 11.13.
+- [v13.1.0] - Postgres update to add `files_granules_cumulus_id_index` to the
+  `files` table may require manual steps depending on load.
+
+#### RDS Phase 3 migration notes
```
[the remark this answers is not in the document]
Comment (Jkovarik) [reply CHANGELOG.md:24]: It looks like it's in the branch, but not in this merge diff.     🤔

File: CHANGELOG.md (hunk 1 above)
Comment (npauzenga) [reply CHANGELOG.md]: Did we maybe mean "minimum supported version"?

File: CHANGELOG.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (npauzenga) [reply CHANGELOG.md]: Yeah, this was a mess. Check out the latest diff. I _think_ everything is in the right spot.

File: CHANGELOG.md [hunk 3]
```diff
@@ -45,9 +63,22 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+
 ### MIGRATION notes
 
-#### RDS Phase 3
+#### PI release version
+
+When updating directly to v16 from prior releases older that V15, please make sure to
+read through all prior release notes.
+
+Notable migration concerns since the last PI release version (11.1.x):
```
Comment (Jkovarik) [inline CHANGELOG.md:75]: Is 11.1.x correct?

Comment (Jkovarik) [reply CHANGELOG.md:75]: Confirmed that 11 was the prior supported version and 11-16 and 15-16 were tested.

**Review by Jkovarik — APPROVED** [review]
Looks good to me, thanks for updating the changelog

File: packages/api/endpoints/schemas.js
Comment (etcart) [inline packages/api/endpoints/schemas.js:1]: we should add something like 
/** @deprecated use accessToken in /api/lib/schemas.js */
in api/models/schemas.js as we move away from using it (I realize it was deprecated before this PR)

**Pull request merged**